content/engineering-practice-layering.md

```diff
@@ -0,0 +1,93 @@
+Title: Engineering Practice Layering
+Date: 2023-03-10
+Category: Posts 
+Tags: code, code-quality, engineering, technical-excellence
+Slug: engineering-practice-layering
+Author: Martin M. Lacey
+Summary: Application Layering - Separation of Concerns Architecture Principle
+
+# What is layering?
+
+Layering is a common construct for identifying discreet areas of interest within an undertaking, so that work can be done in a focused way in each of 
+those areas as well as to highlight the interconnections between each layer.  Typically, each layer has its own specialists that work on the specific
```

Comment:
@MerlinDarkstar what editor are you using? You have paragraph markers in the middle of sentences.

Reply:
I've removed one or two - doesn't seem to make a difference when viewed.  I am using Visual Studio - have the spell checker turned on too but isn't catching any.